### ISSUE TITLE
Confocor FCS file format

### ISSUE THREAD

**Issue by paulmueller** [FCS-analysis/PyCorrFit#37]
http://www.lfd.uci.edu/~gohlke/code/fcsfiles.py.html

Christoph Gohlke wrote a python script for loading Confocor data files. The script works with dictionaries and recurses through the entire file. This might be a better solution than the current (0.7.8) implementation.

**Comment by nashv** [comment]
I am interested, and I see that this already supported in PyCorrFit. However, I have a set of Zeiss Confocor3 


files that simply will not load. A sample is attached.

**Comment by deheyo** [comment]
We probably had the same issue and it was due to the different name of the Auto Correlation detector. So, if you either change the parser in that line or the name within the files (at all measurements). The files loaded afterwards.

https://github.com/FCS-analysis/PyCorrFit/blob/c784b6b655bb213c787348b781e70e79a5c1b94e/pycorrfit/readfiles/read_FCS_Confocor3.py#L86

**Comment by paulmueller** [comment]
Yes, it looks like PyCorrFit is expecting enumerated integer channel names.

@nashv Is it OK with you to put your dataset in the public domain (https://creativecommons.org/choose/zero/). I would need that for testing.
